src/pages/GamePage.js

```diff
@@ -37,10 +37,12 @@ class GamePage extends Component {
   }
 
   initialLocalStorage() {
-    const { name, gravatarEmail } = this.props;
+    const { gravatarEmail, name } = this.props;
     const { assertions, score } = this.state;
     const obj = { player: { name, gravatarEmail, assertions, score } };
-    localStorage.setItem('state', JSON.stringify(obj));
+    const infoPlayer = localStorage.setItem('state', JSON.stringify(obj));
```

Comment:
Salve Filipe, não entendi aqui pq vc setou algo no localStorage colocando isso numa const, e na linha debaixo vc colocou essa const para pegar do localStorage. Para pegarmos algo do localStorage, usamos a `key` do localStorage, ou seja, usamos uma 'string', e quando setamos algo, apenas chamamos a função, sem colocá-la numa const, pois se colocar, retorna undefined. Ficaria dessa forma:
```suggestion
    localStorage.setItem('state', JSON.stringify(obj));
    const getInfoPlayer = JSON.parse(localStorage.getItem('state'));
```